packages/slice-machine/components/ScreenshotChangesModal/index.tsx

```diff
@@ -0,0 +1,205 @@
+        overflow: "hidden",
+        textOverflow: "ellipsis",
+      }}
+    >
+      {slice.model.name}
+    </Heading>
+    <Box>
+      <Box
+        as="ul"
+        sx={{
+          pl: 0,
+          my: 3,
+        }}
+      >
+        {slice.model.variations.map((variation) => {
+          const isSelectedVariation =
+            selectedSliceVariation[0] === slice.model.id &&
+            selectedSliceVariation[1] === variation.id;
+          return (
+            <Box
+              as="li"
+              sx={{
+                listStyle: "none",
+                my: 2,
+                p: 2,
+                cursor: "pointer",
+                borderRadius: "4px",
+                ...(isSelectedVariation && {
+                  bg: "#EDECEE",
+                  fontWeight: "500",
+                }),
+                ":hover": {
+                  bg: "#EDECEE",
+                },
+              }}
+              onClick={() => onSelectVariation(slice.model.id, variation.id)}
+            >
+              <Flex
+                sx={{
+                  alignItems: "center",
+                }}
+              >
+                <VariationIcon isValid={!!slice.screenshots[variation.id]} />
+                <Text sx={{ ml: 2 }}>{variation.name}</Text>
+              </Flex>
+            </Box>
+          );
+        })}
+      </Box>
+    </Box>
+  </>
+);
+
+const ScreenshotChangesModal = ({
+  slices,
+  defaultSelectedSliceVariation = [],
+  onClose,
+}: {
+  slices: ComponentUI[];
+  defaultSelectedSliceVariation?: [string, string] | [];
+  onClose: () => void;
+}) => {
+  const { isOpen } = useSelector((store: SliceMachineStoreType) => ({
+    isOpen: isModalOpen(store, ModalKeysEnum.SCREENSHOTS),
+  }));
+  const [defaultSelectedSliceID, defaultSelectedSliceVariationID] =
+    defaultSelectedSliceVariation;
+
+  const [selectedSliceVariation, setSelectedSliceVariation] = useState<
+    [string, string]
+  >(
+    slices.some(
+      (s) =>
+        s.model.id === defaultSelectedSliceID &&
+        s.model.variations.some((v) => v.id === defaultSelectedSliceVariationID)
+    )
+      ? (defaultSelectedSliceVariation as [string, string])
+      : [slices[0].model.id, slices[0].model.variations[0].id]
+  );
+
+  return (
+    <SliceMachineModal
+      isOpen={isOpen}
+      shouldCloseOnOverlayClick
+      onRequestClose={() => onClose()}
+    >
+      <Card
+        radius={"0px"}
+        bodySx={{ p: 0, bg: "#FFF", position: "relative", height: "100%" }}
+        sx={{ border: "none", height: "60vh" }}
+        Header={({ radius }: { radius: string | number }) => (
+          <Flex
+            sx={{
+              p: "16px",
+              pl: 4,
+              alignItems: "center",
+              justifyContent: "space-between",
+              borderTopLeftRadius: radius,
+              borderTopRightRadius: radius,
+              // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
why the linter error

Reply:
I think this might fix it.
```suggestion
              borderBottom: (t) => `1px solid ${String(t.colors?.borders)}`
```